source/01-LE/02.ptx

```diff
@@ -431,7 +431,7 @@ A matrix is in <term>reduced row echelon form</term> (<term>RREF</term>)<idx>Red
 </li>
 <li> Each pivot is to the right of every higher pivot.
 </li>
-<li> Each term above and below a pivot is zero.
+<li> All terms in each pivot column, besides the pivot itself, are zero.
```

Comment:
So I guess this works, but it introduces confusion for students who mix up columns and rows, which is a surprising (to me) and distressing number of them.

Reply:
Alright, let's try again, and this time I won't make a direct commit before we agree...

Reply:
"Any term that is either above or below a pivot is zero."

Reply:
I thought this would be a simple fix! 🤣. Glad I went through the "trouble" of making it into a pull request so we could have this discussion though.

Reply:
Actually I think I want to rewrite the whole thing to consistently use either "any", "every", or "each"...

Reply:
I think "Any" is confusing and would prefer "Every" or "Each" or "All"

Reply:
- The first nonzero entry of each nonzero row is 1. Call these terms "pivots".
- Each pivot is to the right of every higher pivot.
- Each term that is either above or below a pivot is 0.
- All zero rows are at the bottom of the matrix.

Reply:
Maybe "**We** call these terms 'pivots'", but sounds good!

Reply:
Okay how's that? (I think you may can edit it directly if you want to tweak anything.)

Reply:
I just did it 😬

Reply:
oops you beat me to it (-: